params/config.go

```diff
@@ -30,22 +30,63 @@ import (
 
 // Genesis hashes to enforce below configs on.
 var (
-	MainnetGenesisHash      = common.HexToHash("// todo generate new hash for mainnet") // Mainnet genesis hash to enforce below configs on
-	TestnetGenesisHash      = common.HexToHash("// todo generate new hash for testnet") // Testnet genesis hash to enforce below configs on
+	CypressGenesisHash      = common.HexToHash("// todo generate new hash for cypress") // cypress genesis hash to enforce below configs on
+	BaobabGenesisHash       = common.HexToHash("// todo generate new hash for baobab")  // baobab genesis hash to enforce below configs on
 	AuthorAddressForTesting = common.HexToAddress("0xc0ea08a2d404d3172d2add29a45be56da40e2949")
+	mintingAmount, _        = new(big.Int).SetString("9600000000000000000", 10)
 )
 
 var (
-	// MainnetChainConfig is the chain parameters to run a node on the main network.
-	MainnetChainConfig = &ChainConfig{
-		ChainID: big.NewInt(1),
-		Gxhash:  new(GxhashConfig),
+	// CypressChainConfig is the chain parameters to run a node on the cypress main network.
+	CypressChainConfig = &ChainConfig{
+		ChainID:            big.NewInt(8217),
+		Incompatible1Block: nil,
+		DeriveShaImpl:      2,
+		Governance: &GovernanceConfig{
+			GoverningNode:  common.HexToAddress("0x52d41ca72af615a1ac3301b0a93efa222ecc7541"),
+			GovernanceMode: "single",
+			Reward: &RewardConfig{
+				MintingAmount:          mintingAmount,
+				Ratio:                  "34/54/12",
+				UseGiniCoeff:           true,
+				DeferredTxFee:          true,
+				StakingUpdateInterval:  86400,
+				ProposerUpdateInterval: 3600,
+				MinimumStake:           big.NewInt(5000000),
+			},
+		},
+		Istanbul: &IstanbulConfig{
+			Epoch:          604800,
+			ProposerPolicy: 2,
+			SubGroupSize:   22,
+		},
+		UnitPrice: 25000000000,
 	}
 
-	// TestnetChainConfig contains the chain parameters to run a node on the Ropsten test network.
-	TestnetChainConfig = &ChainConfig{
-		ChainID: big.NewInt(2),
-		Gxhash:  new(GxhashConfig),
+	// BaobabChainConfig contains the chain parameters to run a node on the Baobab test network.
+	BaobabChainConfig = &ChainConfig{
+		ChainID:            big.NewInt(1001),
```

Comment:
Could you use `BaobabNetworkId` instead of`1001`?

Reply:
Thanks, i changed the networkid! Could you please review again?